application/classes/Ushahidi/Repository/Contact.php

```diff
@@ -132,7 +132,8 @@ public function isInTargetedSurvey($contact_id)
     {
         $query = DB::select('targeted_survey_state.contact_id', 'targeted_survey_state.form_id')
             ->from('targeted_survey_state')
-            ->where('contact_id', '=', $contact_id);
+            ->where('contact_id', '=', $contact_id)
+			->and_where('survey_status', '!=', Entity\TargetedSurveyState::SURVEY_FINISHED);
```

Comment:
good catch. should this also include INACTIVE? (or whatever we ended up calling it?)

Reply:
ugh  I might be wrong . .. but I don't think so? , because then the user is in a targeted survey, just another one. Here we just want to filter out the finished ones so users can communicate /send inbounds after they fill  a survey.

Reply:
solved thank you !